dll/win32/imm32/imm.c

```diff
@@ -2933,38 +2933,28 @@ BOOL WINAPI ImmSetCompositionStringW(
 BOOL WINAPI ImmSetCompositionWindow(
   HIMC hIMC, LPCOMPOSITIONFORM lpCompForm)
 {
-    BOOL reshow = FALSE;
-    InputContextData *data = get_imc_data(hIMC);
-
-    TRACE("(%p, %p)\n", hIMC, lpCompForm);
-    if (lpCompForm)
-        TRACE("\t%x, %s, %s\n", lpCompForm->dwStyle,
-              wine_dbgstr_point(&lpCompForm->ptCurrentPos),
-              wine_dbgstr_rect(&lpCompForm->rcArea));
+    DWORD idImeThread, idThread;
+    LPINPUTCONTEXT pIC;
+    HWND hWnd;
 
-    if (!data)
-    {
-        SetLastError(ERROR_INVALID_HANDLE);
+    idImeThread = NtUserQueryInputContext(hIMC, 1);
```

Comment:
cf. remark about this magic "1" from other PR review.

Reply:
Hypothesis: `Imm32QueryInputContext`'s 2nd paramter is same as `Imm32QueryWindow`'s 2nd parameter.